build.gradle

```diff
@@ -30,7 +30,7 @@ subprojects {
     group = "com.sumologic.elasticsearch"
     description = "Elasticsearch Client for Scala that operates against the REST Endpoint"
 
-    version = '7.1.1-SNAPSHOT'
+    version = '7.1.2-SNAPSHOT'
```

Comment:
I think we usually do this version change during the process of releasing. Is version `7.1.1` already released? If not - I think it should stay unchanged.

Reply:
Ack, I will revert version to 7.1.1 (I assume this change is simple enough to not require version bump then).